cdap-docs/developers-manual/source/advanced/application-logback.rst

```diff
@@ -0,0 +1,29 @@
+.. meta::
+    :author: Cask Data, Inc.
+    :copyright: Copyright © 2015 Cask Data, Inc.
+
+.. _application-logback:
+
+====================
+Application Logbacks
+====================
+
+.. highlight:: xml
+
+YARN containers launched by a CDAP application use the default container logback, ``logback-container.xml``,
+packaged with CDAP. This ``logback`` does log rotation once every day at midnight and
+deletes logs older than 14 days. Depending on the use case, the default configuration may be sufficient.
+However, it is also possible to specify a custom ``logback.xml`` for a CDAP application by packaging
+it with the application in the application's ``src/main/resources`` directory.
+The packaged ``logback.xml`` is then used for each container launched by the application.
+
+Writing and packaging a logback.xml
```

Comment:
Do we still need this section, since we already say that we can include `logback.xml` in `src/main/resources` directory above?